requirements-devel.txt

```diff
@@ -128,5 +128,4 @@ zipp==3.17.0
 zope.deprecation==5.0
 zope.interface==6.1
 python-apt @ https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.4.0ubuntu1/python-apt_2.4.0ubuntu1.tar.xz; sys.platform == "linux"
-setuptools<66
```

Comment:
Question.  I know version 66 as a maximum was fixed with https://github.com/snapcore/snapcraft/pull/4490/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L130, but should we still pin it?  Or is setuptools a special case?

Reply:
We don't actually need setuptools with a new enough pip. It's still a build dependency, but that happens in a separate environment. I think this was pip 21? (jammy includes 22)